mslice/scripting/helperfunctions.py

```diff
@@ -0,0 +1,217 @@
+from datetime import datetime
+from mslice.cli.helperfunctions import _function_to_intensity
+from mslice.models.labels import get_recoil_key
+PACKAGES = {
+    'mslice.cli': 'mc',
+    'mslice.plotting.pyplot': 'plt',
+    'matplotlib.colors': 'colors',
+    'numpy': 'np',
+}
+
+
+def cleanup(script_lines):
+    for line in script_lines[-1:]:
+        if line.startswith('Load'):
+            index = script_lines.index(line)
+            return script_lines[index:]
+
+
+def header(plot_handler):
+    from mslice.plotting.plot_window.cut_plot import CutPlot
+    from mslice.plotting.plot_window.slice_plot import SlicePlot
+    statements = ['# Python Script Generated by Mslice on {}\n'.format(datetime.now().replace(microsecond=0))]
+
+    for package in PACKAGES:
```

Comment:
I find this loop a little difficult to read as we have lots of `nots` and `continues`. Should we consider keeping two lists of packages: a base list that is always used and a second list where we can first check the handler and then pull in from the list without needing a loop?

Reply:
@martyngigg Wouldn't we need a loop for the base list?

Reply:
I was thinking of something like
```python
statements.append('\n'.join(COMMON_PACKAGES))
if ...test for whether we have log color bar...:
   statements.append(MPL_COLORS_IMPORT)
if we need numpy:
   statements.append(NUMPY_IMPORT)
```
where `COMMON_PACKAGES` is just a list of import line strings.